flashbot-attack/src/agent.spec.js

```diff
@@ -0,0 +1,127 @@
+const {
+  FindingType,
+  FindingSeverity,
+  Finding,
+} = require('forta-agent');
+const axios = require('axios');
+const {
+  handleBlock,
+  resetLastBlockNumber,
+} = require('./agent');
+
+const from1 = '0xfrom1';
+const to1 = '0xto1';
+const from2 = '0xfrom2';
+const to2 = '0xto2';
+const from3 = '0xfrom3';
+const to3 = '0xto3';
```

Comment:
Probably in the future instead of hard coding these we should start using random address libraries to verify behaviour, can stay like this now